lms/djangoapps/mobile_api/courses/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+Courses API
+"""
```

Comment:
In case you're interested, here's an open source app I ran across that is doing something very similar in terms of querying facebook friends and finding those that have linked accounts:
https://parse.com/tutorials/anypic#follow
They also ended up adding a cache for the list of facebook ids.